src/main/webapp/resources/js/apis/announcements/announcements.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
```

Comment:
Add a file comment as well as comments for the functions

Reply:
Updated in d5647d249f